ffcx/codegeneration/C/integrals.py

```diff
@@ -60,6 +60,7 @@ def generator(ir: IntegralIR, options):
         code["enabled_coefficients"] = "NULL"
 
     code["tabulate_tensor"] = body
+    code["tabulate_tensor_quoted"] = body.replace("\n", '\\n"\n    "')
```

Comment:
Is this always necessary to output? Will increase the code size a lot for large kernels.

Reply:
This only becomes part of the generated code if --cuda is set. Otherwise the format parameter is not used. However, I've updated the code to make the format parameter an empty string if --cuda is not set.